utils/TestUtils.ts

```diff
@@ -98,20 +76,17 @@ export class ValkeyCluster {
                 }
             }
 
-            console.log(command);
             execFile(
                 "python3",
                 [PY_SCRIPT_PATH, ...command.split(" ")],
                 (error, stdout, stderr) => {
                     if (error) {
-                        console.error(stderr);
                         reject(error);
                     } else {
                         const { clusterFolder, addresses: ports } =
                             parseOutput(stdout);
-
                         resolve(
-                            ValkeyCluster.detectVersion().then(
+                            getVersionCallback(ports, cluster_mode).then(
                                 (ver) =>
                                     new ValkeyCluster(ver, ports, clusterFolder)
```

Comment:
After that use new variable `addresses`
```suggestion
                            getVersionCallback(addresses, cluster_mode).then(
                                (ver) =>
                                    new ValkeyCluster(ver, addresses, clusterFolder)
```

Reply:
Updated.